test/VoteManager.js

```diff
@@ -981,122 +722,104 @@ describe('VoteManager', function () {
-        );
-        await voteManager.connect(signers[15]).commit(epoch, commitment2);
-
-        await mineToNextState();
-
-        await voteManager.connect(signers[3]).reveal(epoch, votes,
-          '0x727d5c9e6d18ed15ce7ac8d3cce6ec8a0e9c02481415c0823ea49d847ccb9ddd'); // arguments getvVote => epoch, stakerId, assetId
-
-        await voteManager.connect(signers[15]).reveal(epoch, votes2,
-          '0x727d5c9e6d18ed15ce7ac8d3cce6ec8a0e9c02481415c0823ea49d847ccb9ddd');
-
-        await mineToNextState();
-
-        const stakerIdAcc3 = await stakeManager.stakerIds(signers[3].address);
-        const stakerIdAcc15 = await stakeManager.stakerIds(signers[15].address);
-        const staker = await stakeManager.getStaker(stakerIdAcc3);
-
-        const { biggestStake, biggestStakerId } = await getBiggestStakeAndId(stakeManager, voteManager);
-        const iteration = await getIteration(voteManager, stakeManager, staker, biggestStake);
-        const medians = [100, 200, 300, 400, 500, 600, 700, 800];
-        await blockManager.connect(signers[3]).propose(epoch,
-          medians,
-          iteration,
-          biggestStakerId);
-
-        const ageBefore2 = await stakeManager.getAge(stakerIdAcc15);
-        await mineToNextState(); // dispute
-        await mineToNextState(); // confirm
-        await blockManager.connect(signers[3]).claimBlockReward();
-        await mineToNextState(); // commit
-        epoch = await getEpoch();
-        await voteManager.connect(signers[15]).commit(epoch, commitment1);
-
-        let penalty = toBigNumber(0);
-        let toAdd = toBigNumber(0);
-        let prod = toBigNumber(0);
-        let expectedAgeAfter2 = toBigNumber(ageBefore2).add(10000);
-        expectedAgeAfter2 = expectedAgeAfter2 > 1000000 ? 1000000 : expectedAgeAfter2;
-        for (let i = 0; i < votes2.length; i++) {
-          const tolerance = await collectionManager.getCollectionTolerance(i);
-          const maxVoteTolerance = Math.round(medians[i] + ((medians[i] * tolerance) / BASE_DENOMINATOR));
-          const minVoteTolerance = Math.round(medians[i] - ((medians[i] * tolerance) / BASE_DENOMINATOR));
-
-          prod = toBigNumber(votes2[i]).mul(expectedAgeAfter2);
-          if (votes2[i] > maxVoteTolerance) {
-            toAdd = (prod.div(maxVoteTolerance)).sub(expectedAgeAfter2);
-            penalty = penalty.add(toAdd);
-          } else if (votes2[i] < minVoteTolerance) {
-            toAdd = expectedAgeAfter2.sub(prod.div(minVoteTolerance));
-            penalty = penalty.add(toAdd);
-          }
-        }
-        expectedAgeAfter2 = toBigNumber(expectedAgeAfter2).sub(penalty);
-
-        const ageAfter2 = await stakeManager.getAge(stakerIdAcc15);
-        assertBNEqual(toBigNumber(ageAfter2), (toBigNumber(ageBefore2).add(toBigNumber('10000')).sub(penalty)), 'Incorrect Penalties given');
-      });
+      // it('Correct penalties need to be given even after an asset has been deactivated', async function () {
```

Comment:
Hey @GauravJain9 
Please fix this commented testcase
Following testcase can be used as reference to fix this 
``account 4 should be penalised for incorrect voting in the previous epoch but not account 2 due to asset tolerance``